docs/modules/ROOT/nav.adoc

```diff
@@ -282,6 +282,7 @@
 
 .Appendices
 * xref:release-notes/release-notes-history.adoc[History of Release Notes]
+** xref:release-notes/release-notes-22-0.adoc[Payara Enterprise 5.22.0 Release Notes]
```

Comment:
The latest release notes are always in the General Info section, they shouldn't be duplicated here. I'll remove this line with a direct commit after I merge this PR.

Reply:
Cheers ondrej